.github/workflows/test.yml

```diff
@@ -31,13 +31,9 @@ jobs:
         with:
           version: nightly
 
-      - name: Run Forge build
```

Comment:
not sure why we need to build this for ci testing. saves time to remove this

Reply:
sg